### PR TITLE
docs(forms): remove reference to firefox bug that was fixed

### PR DESCRIPTION
**Pull request by s4m0r4m4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [n/a] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Reference to an outdated bug that was fixed in Firefox.

Issue Number: N/A

## What is the new behavior?
Removed reference to fixed bug in Firefox. Unsure if the IE bug has been addressed, since the link is invalid.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/forms/src/directives/select_control_value_accessor.ts [hunk 1]
```diff
@@ -75,8 +75,7 @@ function _extractId(valueString: string): string {
  * ```
  *
  * **Note:** We listen to the 'change' event because 'input' events aren't fired
- * for selects in Firefox and IE:
- * https://bugzilla.mozilla.org/show_bug.cgi?id=1024350
+ * for selects in IE:
  * https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/4660045/
```
Comment (AndrewKushnir) [inline packages/forms/src/directives/select_control_value_accessor.ts:79]:
```suggestion
 * for selects in IE, see:
 * https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event#browser_compatibility
```

**Review by AndrewKushnir — COMMENTED** [review]
@s4m0r4m4 thanks for keeping the docs up-to-date 👍 

I've left a comment with a proposed change to replace the second link as well. Could you please update it when you get a chance?

Thank you.

**Comment by theBolck** [comment]
MikeJerred:form-add-remove-control-options

**Comment by s4m0r4m4** [comment]
Thanks, I accepted and commited your proposed change to the IE link.

**Review by AndrewKushnir — APPROVED** [review]
Thanks for additional changes @s4m0r4m4 👍 I'm adding this PR to the merge queue.

**Pull request closed without merging**